src/pages/pages/Accounts/show/AccountPage.tsx

```diff
@@ -15,6 +15,9 @@ import placeholderTx from 'assets/images/placeholderTx.svg';
 import { AccountUtils, i18n, NumbersUtils } from 'utils';
 import { LumConstants } from '@lum-network/sdk-javascript';
 import ReactTooltip from 'react-tooltip';
+import AssetsList from '../../../../components/components/AssetsList/AssetsList';
```

Comment:
Careful on the imports paths, fix indexes to use non-relative path to keep consistency